README.md

```diff
@@ -38,7 +38,19 @@ It's a quick read, and it's a great way to introduce yourself to how things work
 
 ## Dependencies
 
-If you are introducing a new dependency, please make sure it's added to:
+The **python-devel**, **krb5-devel** and **gcc** packages are needed in order to be able to
```

Comment:
Although `python-devel` is an alias for `python3-devel`, I think it would be better to have this as:

```suggestion
The **python3-devel**, **krb5-devel** and **gcc** packages are needed in order to be able to
```